ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js

```diff
@@ -0,0 +1,362 @@
+import React, { useState, useEffect } from 'react';
+import { customAxios as axios } from '../../helpers/customAxios';
+import {
+  Container,
+  Heading,
+  Text,
+  Stack,
+  Button,
+  ListItem,
+  Flex,
+  Grid,
+  Spacer,
+  List,
+  Box,
+  Image,
+  HStack,
+  useToast,
+} from '@chakra-ui/react';
+import SermonNotesEditorModal from './SermonNotesEditorModal';
+import { MdOutlineVideoLibrary } from 'react-icons/md';
+import { BiDonateHeart } from 'react-icons/bi';
+import { IoPeopleOutline } from 'react-icons/io5';
+import { IoBookOutline } from 'react-icons/io5';
+import { FaRegCalendarAlt } from 'react-icons/fa';
+
+export default function AdminSermonNotesContainer(props) {
+  const { user } = props;
+  const toast = useToast();
+
+  // This useState is to open the Editor Modal
+  const [isEditorOpen, setIsEditorOpen] = useState(true);
+  const [actionOnEditor, setActionOnEditor] = useState('create');
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [sermonNotesList, setSermonNotesList] = useState([]);
+  const [editSermonNotesData, setEditSermonNotesData] = useState(null);
+
+  const fetchSermonNotes = async () => {
+    try {
+      const { data, status } = await axios.get('/api/sermon-notes-parent/get');
+      if (status !== 200) {
+        throw Error('There was an issue with the request');
+      }
```

Comment:
this if statement is redundant, cuz u alrd put this in try and catch. The try and catch alrd take care of the error here